test/conformance/kuttl/verifyImages/clusterpolicy/standard/keyless-attestations-multiple-subjects-counts-2/01-manifests.yaml

```diff
@@ -27,12 +27,14 @@ spec:
                 issuer: "https://token.actions.githubusercontent.com"
                 rekor:
                   url: https://rekor.sigstore.dev
+                ctlog:
```

Comment:
Will users need to change their policies ?

Reply:
@eddycharly I dont think so, `ignoreSCT` was added in Cosign 2.0 PR: https://github.com/kyverno/kyverno/pull/7248. Which will be released in v1.11. If we add this PR to 1.11, users won't have to change their policies